packages/pancake-uikit/src/widgets/Menu/index.stories.tsx

```diff
@@ -157,11 +114,21 @@ export const Connected: React.FC = () => {
     </BrowserRouter>
   );
 };
+export const Connected = ConnectedTemplate.bind({});
+Connected.args = defaultProps;
 
 export const NotConnected: React.FC = () => {
   return (
     <BrowserRouter>
-      <Menu isDark={false} toggleTheme={noop} langs={langs} setLang={noop} currentLang="EN" links={links}>
+      <Menu
+        isDark={false}
+        toggleTheme={noop}
+        langs={langs}
+        setLang={noop}
+        currentLang="EN"
+        links={links}
+        subLinks={subLinks}
+      >
```

Comment:
I want to move away from having children the body of the page. The menu should be an isolated component.

Reply:
For example
```
<Menu />
<RestOfTheApp />
```

Reply:
I agree, will move to that.

Reply:
I think I'll address this in a dedicated PR, it will require some more changes.